src/util/createServerRootMixin.js

```diff
@@ -235,7 +242,11 @@ function augmentInstantSearch(instantSearchOptions, searchClient, indexName) {
 }
 
 export function createServerRootMixin(instantSearchOptions = {}) {
-  const { searchClient, indexName } = instantSearchOptions;
+  const {
+    searchClient,
+    indexName,
+    $cloneComponent = defaultCloneComponent,
```

Comment:
when does $cloneComponent gets passed to `instantSearchOptions`

Reply:
could be a separate argument for Vue InstantSearch-specific arguments too, but didn't see much use yet. Can be changed!

Reply:
yeah we could see it as `type Options = InstantSearchOptions & VueInstantSearchOptions`, but no strong opinion, so I'm also okay with having it as in the second parameter.